src/restrictededitingmodeediting.js

```diff
@@ -311,34 +309,31 @@ function getCommandExecuter( editor, commandName ) {
 	};
 }
 
-// Helper for handling custom keydown behaviour.
-export function onKeyDown( editor, eventInfo, evtData ) {
+// Helper for handling Ctrl+A keydown behaviour.
+function onSelectAll( editor, data, cancel ) {
```

Comment:
Use `model` instead of `editor` one line less in code below :D

Reply:
As we solved this on Slack, I'd stick to `editor` rather then `model` because it's needed to get marker position in `getMarkerAtPosition(editor, position)` few lines below.